package.json

```diff
@@ -31,6 +31,7 @@
     "watch": "node tasks/watch.js",
     "lint": "eslint --version && eslint .",
     "lint-fix": "eslint . --fix || true",
+    "no-new-func": "eslint --no-ignore --no-eslintrc --rule '{no-new-func: error}' dist/plotly-finance.js dist/plotly-basic.js dist/plotly-cartesian.js dist/plotly-geo.js dist/plotly-mapbox.js",
```

Comment:
Nice simple command 🌟 I wonder though, is it possible to express this as a list of bundles to _exclude_  - ie "test everything in dist/*js except *.min.js and plotly.js and plotly-gl3d.js and ..." that way over time we simply remove exclusions from the list, and any new bundle we create is automatically included?

Reply:
Done in https://github.com/plotly/plotly.js/pull/5383/commits/c611031939f10c64d1afdf2dc9bc0f0207989361.